app/views/clones/new.html.erb

```diff
@@ -6,12 +6,12 @@
       <%= hidden_field_tag :project_hash_id, @project.hash_id  %>
       <p class="mt2">
         <strong>Who will be working on this project?</strong>
-        <%= text_field_tag :students, "" , class: "block col-12 field"  %>
+        <%= text_field_tag :students, @clone.students, class: "block col-12 field"  %>
         <em>e.g. <strong>Leslie Knope</strong> if solo; <strong>Leslie Knope and Duke Silver.</strong> if a team project.</em>
       </p>
       <p class="mt2">
         <strong>We'll send you an email when everything is finished. Where would you like us to send to?</strong>
-        <%= text_field_tag :email, "" , class: "block col-12 field"  %>
+        <%= text_field_tag :email, nil, class: "block col-12 field"  %>
```

Comment:
it'd be nice to populate this with `current_user.email` if we have it (it should be nil/empty otherwise), I think we only have one user in the production system who has a blank email address.

Reply:
I like that addition 👍